AjaxFetch/js/show.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * show users
+ * @param {string} parent
+ */
+function user(parent) {
+  $.each(window.values.users, (userId, user) => {
+    let posts = $(`<div style="display: none"></div>`);
```

Comment:
Don't use inline styles. For hide this element, you can use **.hide() ** method of jQuery